packages/angular-display/src/lib/pipe/is-display.pipe.ts

```diff
@@ -0,0 +1,57 @@
+import {ChangeDetectorRef, OnDestroy, Pipe, PipeTransform} from '@angular/core';
+import {Subject, switchMap, Unsubscribable} from 'rxjs';
+import {distinctUntilChanged} from 'rxjs/operators';
+import {IsDisplayService} from '../service/is-display.service';
+import {parseOption, ValidOption} from '../shared/display-option.adapter';
+
+/**
+ * Converts the provided breakpoint-query to a boolean value depending on the current display size.
+ * For the breakpoint-queries passed to this directive please consult {@link provideIsDisplay}.
+ *
+ * Example:
+ * ```html
+ * <span>
+ *     {{ 'small' | clfIsDisplay }}
+ * </span>
+ * ```
+ *
+ * The value automatically updates whenever the display size changes.
+ *
+ * @see {@link ValidOption}
+ * @see {@link provideIsDisplay}
+ */
+@Pipe({
+    name: 'clfIsDisplay',
+    standalone: true,
+    pure: false,
+})
+export class IsDisplayPipe implements PipeTransform, OnDestroy {
+    private isDisplay = false;
+    private option$ = new Subject<ValidOption>();
```

Comment:
```suggestion
    private option$ = new ReplaySubject<ValidOption>(1);
```

Reply:
why would I use a ReplaySubject? The Subject is subscribed to in the `constructor` and unsubscribed in `ngOnDestroy`.

Reply:
I have helped quite some projects fixing bugs where the cause was using `Subject` instead of `ReplaySubject` or `BehaviorSubject`. My recommendation would be to always use any of these two and not a plain `Subject` unless we explicitly want to ignore all values emitted in the past.

Reply:
I still don't see the point in this case, but since you insist I will change it.